fireant/slicer/widgets/pandas.py

```diff
@@ -121,7 +121,7 @@ def pivot_data_frame(self, data_frame, pivot=(), transpose=False):
             .fillna(value='')
 
     def sort_data_frame(self, data_frame):
-        if not self.sort:
+        if not self.sort or len(data_frame) == 1:
```

Comment:
What's the implication of having len(data_frame) == 1 here? Perhaps it'd be better to have a variable named has_some_property or is_some_property, which would be used in this if, so that people know why len(data_frame) == 1 must return the dataframe.

Reply:
This is quite literal. If the data frame has one row, then sorting it would not yield any effect. I can add a comment.